LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan/DevAddr.cs

```diff
@@ -27,6 +27,15 @@ namespace LoRaWan
 
         public DevAddr(uint value) => this.value = value;
 
+        public DevAddr(int networkId, int networkAddress)
+            : this((networkId, networkAddress) switch
+            {
+                ( > 0x7F, _) => throw new ArgumentException(null, nameof(networkId)),
+                (_, > (int)NetworkAddressMask) => throw new ArgumentException(null, nameof(networkAddress)),
+                _ => ((uint)networkId << 25) | (uint)networkAddress
```

Comment:
I think we can safely use unchecked conversions at this point:

```suggestion
                var (id, addr) => unchecked(((uint)id << 25) | (uint)addr)
```